gravitee-apim-portal-webui-next/src/scss/material-design-overrides.scss

```diff
@@ -29,3 +29,13 @@ mat-card {
 mat-form-field {
   --mdc-filled-text-field-container-color: var(--mdc-outlined-card-container-color);
 }
+
+.search-input {
+  .mdc-text-field--filled {
+    border-radius: 28px;
+  }
+
+  .mdc-line-ripple {
+    display: none;
+  }
+}
```

Comment:
Seeing as this is specific to a component, it may be better that it stays in the component scss. (Left a comment with a fix for that....)

Reply:
Done